scripts/nbconverted/explore_relationship_AandB_pt2.py

```diff
@@ -0,0 +1,682 @@
+
+# coding: utf-8
+
+# # Explore simulated relationship (part 2)
+# 
+# This notebook is using simulated data generated from [main_Pa_sim_enhance_AtoB](main_Pa_sim_enhance_AtoB.ipynb).  This notebook input raw Pseudomonas gene expression data from the Pseudomonas compendium referenced in [ADAGE](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5069748/) paper and added a strong nonlinear signal.  This signal assigned a set of genes to group A and a set of genes to group B.  If the expression of genes in group A exceeded some threshold then the genes in group B were upregulated.  
+# 
+# This notebook is extending from the exploration performed in [explore_relationship_AandB](explore_relationship_AandB.ipynb).  In this notebook we determined that the modeled/predicted gene expression data between A and B (i.e. after applying a linear transformation in the latent space and decoding) is a mostly linear relationship.  We assume that this means that the *decoder* is learning this linear relationship.  So now we want to determine what the *encoder* is learning. 
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import pandas as pd
+import numpy as np
+import random
+import glob
+import seaborn as sns
+from keras.models import model_from_json, load_model
+from functions import utils
+import matplotlib.pyplot as plt
+from matplotlib_venn import venn2
+
+from numpy.random import seed
+randomState = 123
+seed(randomState)
+
+
+# In[2]:
+
+
+# Run notebook to generate simulated data
+#%run ./main_Pa_sim_enhance_AtoB.ipynb
+
+
+# In[3]:
+
+
+# Load 
+base_dir = os.path.dirname(os.getcwd())
+analysis_name = 'sim_AB_2775_300_v2'
+
+sim_data_file = os.path.join(base_dir, "data", analysis_name, "train_model_input.txt.xz")
+A_file = os.path.join(base_dir, "data", analysis_name, "geneSetA.txt")
+B_file = os.path.join(base_dir, "data", analysis_name, "geneSetB.txt")
+
+offset_vae_file = os.path.join(os.path.dirname(os.getcwd()), "encoded", analysis_name, "offset_latent_space_vae.txt")
+
+weight_file = os.path.join(os.path.dirname(os.getcwd()), "data", analysis_name, "VAE_weight_matrix.txt")
+
+model_encoder_file = glob.glob(os.path.join(base_dir, "models", analysis_name, "*_encoder_model.h5"))[0]
```

Comment:
Would prefer these long lines to be broken up into a version with more white space:
```python
glob.glob(
  os.path.join(
  base_dir, "models", 
  analysis_name, "*_encoder_model.h5"
  )
)[0]
```